UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -19,7 +19,8 @@ class BTPayPalClient_Tests: XCTestCase {
         mockAPIClient.cannedResponseBody = BTJSON(value: [
             "paymentResource": ["redirectUrl": "http://fakeURL.com"]
         ])
-        payPalClient = BTPayPalClient(apiClient: mockAPIClient, universalLink: URL(string: "https://www.paypal.com")!)
+        payPalClient = BTPayPalClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn", universalLink: URL(string: "https://www.paypal.com")!)
```

Comment:
Lets continue to use the `development_testing_integration_merchant_id` tokenization key. We can likely extract this into a constant in this file as well.